swyft/types.py

```diff
@@ -4,14 +4,15 @@
 
 import numpy as np
 import torch
+import zarr
 
 # If you add a type, don't forget to update autodoc_type_aliases in /docs/source/config.py
 
 # General defintions
 PathType = Union[str, Path]
 Device = Union[torch.device, str]
 Tensor = torch.Tensor
-Array = Union[np.ndarray, torch.Tensor]
+Array = Union[np.ndarray, torch.Tensor, zarr.Array]
```

Comment:
I believe that `zarr.Array` doesn't allow the same indexing as `ndarray` and `Tensor`. e.g. `array[:, [1, 2, 3]]`.

In that case, I'd like to keep these distinct.